Modules/gcmodule.c

```diff
@@ -2203,6 +2203,13 @@ PyObject_GC_UnTrack(void *op_raw)
     }
 }
 
+/* Test if an object has a GC head */
```

Comment:
Nitpick: usually, I prefer to not document a function twice: only keep the comment in the header file. It avoids inconsistencies if one is updated, but not the other.

Reply:
make sense.